src/Symfony/Component/HttpClient/Tests/EventSourceHttpClientTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpClient\Tests;
+
+require __DIR__.'/../../../../../vendor/autoload.php';
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\HttpClient\EventSourceHttpClient;
+use Symfony\Component\HttpClient\HttpClient;
+
+// class EventSourceHttpClientTest extends TestCase
+// {
+//     public function testMessageClient()
+//     {
+//
+//     }
+// }
+
+$es = new EventSourceHttpClient(HttpClient::create());
+
+$response = $es->request('GET', 'http://localhost:8080/events');
```

Comment:
We already have Vulcain in the CI to test HttpClient, you could add Mercure, it uses all the spec capabilities.